core/modules/loader/udpTest.cc

```diff
@@ -0,0 +1,542 @@
+// -*- LSST-C++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2018 AURA/LSST.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+// System headers
+#include <iostream>
+#include <boost/asio.hpp>
```

Comment:
Why aren't you using boost unit test framework for this?

Reply:
This is not a unit test. Unit tests have the following names `test<class>.cc`.

Reply:
I intend to switch over to the unit framework, but many of these are more integration tests than unit tests and are resource intensive. Splitting out the unit tests from the integration tests is something I'd rather do in a separate ticket.

Reply:
DM-16649